src/main/groovy/edu/wpi/first/gradlerio/wpi/dependencies/WPIDependenciesPlugin.groovy

```diff
@@ -48,6 +48,11 @@ class WPIDependenciesPlugin implements Plugin<Project> {
             }
         } as Action<Task>)
 
+        project.tasks.register("vendordep", VendorDepTask, { VendorDepTask task ->
+            task.group = "GradleRIO"
+            task.description = "Install vendordep JSON file from URL"
+        } as Action<VendorDepTask>)
+
```

Comment:
You dont need the as here. Instead, do like the below one does, and close the register block after the `VendorDepsTask` part. Groovy will then automatically take the closure afterwards as the 3rd parameter.

Reply:
Fixed?

Reply:
Nope.

```
project.tasks.register("vendordep", VendorDepTask) { VendorDepTask task ->
            task.group = "GradleRIO"
            task.description = "Install vendordep JSON file from URL"
        }
```

Reply:
So Kotlin got the last-lambda-parameter-as-block syntax from groovy... 
And is it fixed now?